src/static_handler.rs

```diff
@@ -105,9 +103,10 @@ impl Handler for Static {
         if requested_path.should_redirect(&metadata, req) {
             // Perform an HTTP 301 Redirect.
             let mut redirect_path = match req.extensions.get::<OriginalUrl>() {
-                None => &req.url,
-                Some(original_url) => original_url,
-            }.clone();
+                                        None => &req.url,
+                                        Some(original_url) => original_url,
+                                    }
+                                    .clone();
```

Comment:
Yeah. I'm not sure that I agree with rustfmt here. There may be a parameter to stop it from doing this, but I'm not seeing a clear choice. I'll change it back.